lang/java/compiler/src/main/java/org/apache/avro/compiler/schema/CloningVisitor.java

```diff
@@ -1,4 +1,18 @@
-
+/*
+ * Copyright 2015 The Apache Software Foundation.
```

Comment:
I think, the copyright + year part is not necessary. Most of the other java files (e.g. `org.apache.avro.Schema`) do not contain it.

Reply:
Should I remove it?

Reply:
Yes please. Just make it the same as all other .java files.